lib/pages/new_group/new_group_chat_info.dart

```diff
@@ -1,429 +1,360 @@
-                      builder: (context, snapshot) {
-                        if (snapshot.hasData) {
-                          final maxMediaSize = snapshot.data!.mUploadSize;
-                          return Text(
-                            L10n.of(context)!
-                                .maxImageSize(maxMediaSize!.bytesToMB()),
-                            style: Theme.of(context)
-                                .textTheme
-                                .bodySmall
-                                ?.copyWith(
-                                  color:
-                                      LinagoraRefColors.material().neutral[40],
-                                ),
-                          );
-                        } else {
-                          return const SizedBox.shrink();
-                        }
-                      },
-                    ),
-                    const SizedBox(height: 32),
-                    _buildGroupNameTextField(context),
-                    const SizedBox(height: 16),
-                    _EncryptionSettingTile(
-                      enableEncryptionNotifier:
-                          newGroupController.enableEncryptionNotifier,
-                      onChanged: (value) {
-                        newGroupController.toggleEnableEncryption();
-                      },
-                    ),
-                  ],
-                ),
-              ),
-            ),
-          ];
-        },
-        body: Padding(
-          padding: NewGroupChatInfoStyle.padding,
-          child: Column(
-            mainAxisSize: MainAxisSize.min,
-            children: [
-              Expanded(
-                child: ExpansionParticipantsList(
-                  newGroupController: newGroupController,
-                  contactsList: contactsList,
-                ),
-              ),
-            ],
-          ),
-        ),
+  State<StatefulWidget> createState() => NewGroupChatInfoController();
+}
+
+class NewGroupChatInfoController extends State<NewGroupChatInfo>
+    with CommonMediaPickerMixin, SingleImagePickerMixin {
+  final enableEncryptionNotifier = ValueNotifier(false);
+  final haveGroupNameNotifier = ValueNotifier(false);
+  final createRoomStateNotifier =
+      ValueNotifier<Either<Failure, Success>>(Right(CreateNewGroupInitial()));
+  final uploadContentInteractor = getIt.get<UploadContentInteractor>();
+  final uploadContentWebInteractor =
+      getIt.get<UploadContentInBytesInteractor>();
+  final createNewGroupChatInteractor =
+      getIt.get<CreateNewGroupChatInteractor>();
+  final groupNameTextEditingController = TextEditingController();
+  final avatarAssetEntityNotifier = ValueNotifier<AssetEntity?>(null);
+  final avatarFilePickerNotifier = ValueNotifier<FilePickerResult?>(null);
+
+  final groupNameFocusNode = FocusNode();
+  StreamSubscription? createNewGroupChatInteractorStreamSubscription;
+
+  final responsiveUtils = getIt.get<ResponsiveUtils>();
```

Comment:
can you move it to a style file ?

Reply:
done

Reply:
I think i will keep it here, so that controller don't depends on style class